### ISSUE TITLE
Fewer data for compared methods on Omniglot

### ISSUE THREAD

**Issue by Kostis-S-Z** [khurramjaved96/mrcl#6]
Hello K. Javed,

I read your paper and was very excited with the idea that you proposed because of its intuitiveness. Thank you for your work and for open-sourcing the code. One thing I found is that it seems that the methods **baseline** and **SR-NN** are using fewer samples to pretrain the models than  MRCL / OML. Can you confirm if this is indeed the case, if there is a reason for this or if its a bug to fix it? If its a bug, it is fairly straight-forward how to fix it.

Specifically, it seems that the amount of classes selected to pretrain the models are hard-coded. This can be found here:

_**baseline_pretraining.py**_ -> line 40
`iterator = torch.utils.data.DataLoader(utils.remove_classes_omni(dataset, list(range(900))), batch_size=256, shuffle=True, num_workers=1)`

_**baseline_srnn.py**_ -> line 37
`iterator = torch.utils.data.DataLoader(utils.remove_classes_omni(dataset, list(range(900))), batch_size=256, shuffle=True, num_workers=1)`

whereas in _**mrcl_classification.py**_ -> line 27
`args.classes = list(range(963))`

this causes the baseline and the SR-NN to be trained with 900 * 15 = 13.500 samples whereas for MRCL / OML with 963 * 20 = 19.260 samples. You can also confirm this is the case by just adding a counter in the for loop (`count_samples += list(img.shape)[0]`) of the iterator and see the difference of samples.

**Comment by khurramjaved96** [comment]
Hi, 

Thanks for the comment. This was indeed the case when we submitted the paper for review -- it was an oversight. We later fixed it and updated the results in the paper. This is why the graphs in the repo are slightly different than those in the current version of the paper for the Omniglot benchmark, but the differences are small. 
 
I haven't gotten around to pushing the code that we used for the camera-ready version to github. I'm planning to do that soon.
